Containerfile

```diff
@@ -13,6 +13,8 @@ RUN microdnf -y install \
 
 COPY . .
 
+ENV GOPROXY="https://proxy.golang.org,direct"
```

Comment:
Note that we also have a `cachito_athens_url` in the config that would override `GOPROXY`, but this still seems like a good default

Reply:
Oh, good point, I had forgotten about it.